python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -76,8 +78,10 @@ def __init__(self,
                  process_runner=subprocess,
                  update_interval_s=AUTOSCALER_UPDATE_INTERVAL_S,
                  prefix_cluster_info=False,
-                 event_summarizer=None):
+                 event_summarizer=None,
+                 log_dir="/tmp"):
```

Comment:
can we not give a default value here?

Reply:
default value is to avoid refactoring dozens of test cases